requirements.txt

```diff
@@ -1 +1 @@
-requests
+httpx
```

Comment:
A note from the deployment prospective: we need to make sure we package `httpx` in our conda-channel, as https://anaconda.org/search?q=httpx only provides a conda-forge version:
![image](https://user-images.githubusercontent.com/13209176/81731278-b6067500-945c-11ea-9229-5fb043b7c3a7.png)

Reply:
I create httpx PR here https://github.com/nsls-ii-forge/httpx-feedstock/pulls